datamade/how-to
# Configure Django cookiecutter for production mode

## Overview

This PR adds in changes that configure Django compressor and pytest/pytest-django to work properly in production mode.

Handles #283 


## Testing Instructions

* Create a sample project with the cookiecutter
* Build and bring up the app with `docker-compose build` and `docker-compose up`
* Verify that the web pages are accessible
* Add a simple React component to the homepage and verify that it renders properly
